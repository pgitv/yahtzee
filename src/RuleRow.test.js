import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import RuleRow from './RuleRow';

it('renders shallowly without crashing', function() {
  shallow(<RuleRow />);
});

it('passes snapshot test without a click', function() {
  let wrapper = shallow(<RuleRow />);
  let serialized = toJson(wrapper);
  expect(serialized).toMatchSnapshot();
});
