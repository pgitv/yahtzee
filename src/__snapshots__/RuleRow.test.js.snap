// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`passes snapshot test without a click 1`] = `
<tr
  className="RuleRow RuleRow-active"
>
  <td
    className="RuleRow-name"
  />
  <td
    className="RuleRow-score"
  />
</tr>
`;
